src/AzureExtension/DevBox/Models/DevBoxCreationParameters.cs

```diff
@@ -15,16 +15,24 @@ public class DevBoxCreationParameters
 
     public string PoolName { get; set; } = string.Empty;
 
-    public string DevBoxName { get; set; } = string.Empty;
+    public string NewEnvironmentName { get; set; } = string.Empty;
```

Comment:
Actually this is used because in Dev Home we have no way of know the name of the compute system  the user is attempting to create based on the CreateComputeSystemOperation. See this comment in Dev Home where I explain it a bit more: https://github.com/microsoft/devhome/blob/4559792e7c6c0e62a4da32bbd32011c4fe4c4a20/common/Environments/Models/CreateComputeSystemOperation.cs#L69